mmhuman3d/models/architectures/mesh_estimator.py

```diff
@@ -248,13 +240,9 @@ def run_registration(
         # it comes from SMPL
         gt_out = self.body_model_train(
             betas=gt_betas, body_pose=gt_pose, global_orient=gt_global_orient)
-        if num_keypoints == 49:
-            gt_model_joints = gt_out['joints']
-            gt_vertices = gt_out['vertices']
-        else:
-            gt_model_joints = gt_out['joints'][:, 25:, :]
-            gt_vertices = gt_out['vertices']
-        # TODO: add joint mask
+        assert num_keypoints == 49
```

Comment:
Let's leave a `TODO` comment to indicate future modification that allows for more keypoint conventions.

Reply:
Done